node/impl/full/state.go

```diff
@@ -2043,12 +2043,23 @@ func (a *StateAPI) StateGetNetworkParams(ctx context.Context) (*api.NetworkParam
 		return nil, err
 	}
 
+	genBlock, err := a.Chain.GetGenesis(ctx)
+	if err != nil {
+		return nil, xerrors.Errorf("getting genesis: %w", err)
+	}
+
+	genesisTimestamp := uint64(0)
+	if genBlock != nil {
+		genesisTimestamp = genBlock.Timestamp
+	}
```

Comment:
```suggestion
	var genesisTimestamp uint64
	if genBlock, err := a.Chain.GetGenesis(ctx); if err != nil {
		return nil, xerrors.Errorf("getting genesis: %w", err)
	} else if genBlock != nil {
		genesisTimestamp = genBlock.Timestamp
	}
```

this whole block could be shortened